Signin.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Sign in</title>
+</head>
+<style>
+    .signin{
+        background-color: cadetblue;
+    }
+    .signin_1{
+        text-align: center;
+        padding: 30px;
+        margin: 100px;
+    }
+</style>
+<body class="signin">
+<div class="sigin_1">
+        <h2>Sign in<h2>
+    <from action="/signin">
+        <div>
+            <input placeholder="Email" name="123@cits5505.com" />

```

Comment:
Can we change the name attribute to emailId or something having just text instead of '123@cits5505.com"